src/statsmodel.jl

```diff
@@ -189,7 +189,19 @@ function Base.show(io::IO, model::TableModels)
         println(io, model.mf.f)
         println(io)
         println(io,"Coefficients:")
-        show(io, ct)
+        println(io, ct)
+        if model isa TableRegressionModel
+            println(io)
+            println("R²: ", round(r2(model), sigdigits=4))
+            try
+                fstat = fstatistic(model)
+                println(io, fstat)
+            catch e
+                if !(isa(e, MethodError) && e.f == fstatistic)
+                    rethrow(e)
+                end
+            end
```

Comment:
general speaking `if x isa Y` is a sign of non idiomatic Julia code. I would do something like 
```julia
_show_fit_stats(io::IO, model::TableModel) = nothing

function _show_fit_stats(io::IO, model::TableRegressionModel)
# stuff
end
```

and add `_show_fit_stats(io, model)` to the `show` method